Tasks/CmdLineV2/typings.json

```diff
@@ -2,6 +2,7 @@
   "name": "vsts-cmdline-task",
   "dependencies": {},
   "globalDependencies": {
+    "mocha": "registry:dt/mocha#2.2.5+20160720003353",
```

Comment:
We should get rid of typings and replace with @ types. Strangely I thought we already had everywhere.

Reply:
For some reason it really doesn't like me changing to `@types`, especially for node. Are you ok merging here and we can follow up on that later? There's also a fair amount of tasks that use typings - why is it a problem?

Reply:
Let's file an issue to do it so we can track.

Reason: Typings was deprecated a long time ago.

Reply:
Done - https://github.com/microsoft/azure-pipelines-tasks/issues/11620